### ISSUE TITLE
0.54.0: Too many files open

### ISSUE THREAD

**Issue by tagliala** [sds/haml-lint#481]
I have a decently large repository with 555 haml files and 0.54.0 does not work anymore

```
$ haml-lint
Too many open files @ rb_sysopen - app/views/messages/_message.html.haml
~/dev/haml-lint/lib/haml_lint/runner.rb:147:in `initialize'
~/dev/haml-lint/lib/haml_lint/runner.rb:147:in `new'
~/dev/haml-lint/lib/haml_lint/runner.rb:147:in `block in extract_applicable_sources'
~/dev/haml-lint/lib/haml_lint/runner.rb:146:in `map'
~/dev/haml-lint/lib/haml_lint/runner.rb:146:in `extract_applicable_sources'
~/dev/haml-lint/lib/haml_lint/runner.rb:24:in `run'
~/dev/haml-lint/lib/haml_lint/cli.rb:114:in `scan_for_lints'
~/dev/haml-lint/lib/haml_lint/cli.rb:58:in `act_on_options'
~/dev/haml-lint/lib/haml_lint/cli.rb:25:in `run'
~/dev/haml-lint/bin/haml-lint:7:in `<top (required)>'
```

Bisected to 5a80d11f6f2a00b6a717e5ee7f7b5e7d67c95f8a (#478)

0.53.0 works just fine

**Comment by sds** [comment]
Thanks for the report!

@benmelz can you have a look?

**Comment by benmelz** [comment]
Can do

**Comment by sds** [comment]
Fix released in [0.55.0](https://github.com/sds/haml-lint/releases/tag/v0.55.0). Thanks @tagliala for the report and @benmelz for the quick fix.

**Comment by tagliala** [comment]
Thank you both for the quick fix and release.

I can confirm that 0.55.0 fixes the issue I was having
